### ISSUE TITLE
Panic error after starting node

### ISSUE THREAD

**Issue by zjsunzone** [PlatONnetwork/PlatON-Go#1322]
Hi there,

Panic error after starting node.

#### System information

PlatON version: `feature/optimization-of-storage`
OS & Version: `Linux`
Commit hash : `fd020a0b4e82af8f3`

#### Expected behavior

Normal

#### Actual behaviour

Error

#### Steps to reproduce the behavior

Start node.

#### Backtrace

````
INFO [04-08|19:24:38.313|consensus/cbft/wal_bridge.go:143]          Success to update chainState             commitState="[blockNumber:20180, blockHash:0x87b45869fef9a0f72fa7eb07f4ef9ca88d7b4cf9f08167945a24fd2224fd9296, qcNumber:20180, qcHash:0x87b45869fef9a0f72fa7eb07f4ef9ca88d7b4cf9f08167945a24fd2224fd9296]" lockState="[blockNumber:20181, blockHash:0x28bc81ad0fa198cfc06f52b3c3cbb99972404bc51fbd106cdfd45fdc3aa52b20, qcNumber:20181, qcHash:0x28bc81ad0fa198cfc06f52b3c3cbb99972404bc51fbd106cdfd45fdc3aa52b20]" qcState="[blockNumber:20182, blockHash:0x5f1c27fa1157c28581967de91d2f6b5b3d007ffbf791b50ad3504a4b0911f39d, qcNumber:20182, qcHash:0x5f1c27fa1157c28581967de91d2f6b5b3d007ffbf791b50ad3504a4b0911f39d]" elapsed=117.814µs
panic: statedb readonly

goroutine 104 [running]:
github.com/PlatONnetwork/PlatON-Go/core/state.(*StateDB).setStateObject(...)
	/PlatON-Go/build/_workspace/src/github.com/PlatONnetwork/PlatON-Go/core/state/statedb.go:663
github.com/PlatONnetwork/PlatON-Go/core/state.(*StateDB).getStateObject(0xc000186c60, 0x10, 0x0, 0x5000000, 0x4ed4)
	/PlatON-Go/build/_workspace/src/github.com/PlatONnetwork/PlatON-Go/core/state/statedb.go:657 +0x578
github.com/PlatONnetwork/PlatON-Go/core/state.(*StateDB).GetState(0xc000186c60, 0x10, 0x0, 0x5000000, 0x2083ffc, 0x7, 0x7, 0x0, 0x0, 0x0)
	/PlatON-Go/build/_workspace/src/github.com/PlatONnetwork/PlatON-Go/core/state/statedb.go:334 +0xa6
github.com/PlatONnetwork/PlatON-Go/x/gov.ListActiveVersion(0x16c5ac0, 0xc000186c60, 0x0, 0x0, 0x0, 0x0, 0x0)
	/PlatON-Go/build/_workspace/src/github.com/PlatONnetwork/PlatON-Go/x/gov/gov_db.go:476 +0x7b
github.com/PlatONnetwork/PlatON-Go/x/gov.GetCurrentActiveVersion(0x16c5ac0, 0xc000186c60, 0xb96f567548f000ce)
	/PlatON-Go/build/_workspace/src/github.com/PlatONnetwork/PlatON-Go/x/gov/gov.go:77 +0x39
github.com/PlatONnetwork/PlatON-Go/core.(*BlockChain).WriteBlockWithState(0xc003df2000, 0xc00f09f280, 0x21b0688, 0x0, 0x0, 0xc000186c60, 0x0, 0x0, 0x0)
	/PlatON-Go/build/_workspace/src/github.com/PlatONnetwork/PlatON-Go/core/blockchain.go:972 +0x1a16
github.com/PlatONnetwork/PlatON-Go/miner.(*worker).resultLoop(0xc00000c3c0)
	/PlatON-Go/build/_workspace/src/github.com/PlatONnetwork/PlatON-Go/miner/worker.go:700 +0xd21
created by github.com/PlatONnetwork/PlatON-Go/miner.newWorker
	/PlatON-Go/build/_workspace/src/github.com/PlatONnetwork/PlatON-Go/miner/worker.go:252 +0x859
````

**Comment by TraceBundy** [comment]
The reason is that the logic of the forked code is left

**Comment by benbaley** [comment]
fixed in #1321
